package.json

```diff
@@ -16,7 +16,8 @@
   "gypfile": true,
   "scripts": {
     "install": "node-gyp rebuild",
-    "pretest": "jscs . && jshint .",
+    "instrument": "node node_modules/tap/node_modules/nyc/node_modules/istanbul/lib/cli.js instrument --output backend.cov --no-compact --preserve-comments backend",
+    "pretest": "jscs . && jshint . && npm run instrument",
     "test": "tap test/lab/*.test.js test/*.test.js"
```

Comment:
Since I am going to change build-framework anyways, I prefer to inject these two extra args there.